src/sentry/issues/priority.py

```diff
@@ -16,13 +16,13 @@
     from sentry.models.group import Group
 
 
-PRIORITY_LEVEL_TO_STR: dict[int, str] = {
+PRIORITY_VALUE_TO_STR: dict[int, str] = {
```

Comment:
instead of defining dictionaries for these, what do you think about putting class methods on the `PriorityLevel` enum object, similar to

https://github.com/getsentry/sentry/blob/a1ba3d0f24f9975b77c173324d67535e90b1048e/src/sentry/utils/outcomes.py#L25
?

could make it easier to maintain, as you wouldn't have to add new lines here if we end up adding more priorities, and you can handle when an invalid priority comes in right in the method, instead of the caller

Reply:
That's a great idea, thank you!